arm-storage/2017-06-01/swagger/storage.json

```diff
@@ -638,11 +638,16 @@
         },
         "keySource": {
           "type": "string",
-          "description": "The encryption keySource (provider). Possible values (case-insensitive):  Microsoft.Storage, Microsoft.KeyVault",
+          "description": "The encryption keySource (provider). Possible values (case-insensitive):  Microsoft.Storage, Microsoft.Keyvault",
```

Comment:
minor: Although the provide namespace are case-insensitive, we should name it as modeled in KeyValue's [swagger ](https://github.com/Azure/azure-rest-api-specs/blob/master/arm-keyvault/2016-10-01/swagger/keyvault.json#L19) for it's provider namespace. 
It'd be good to use `Microsoft.KeyVault` instead `Microsoft.Keyvault` for consistency across Azure services